src/mds/MDCache.cc

```diff
@@ -8358,6 +8358,40 @@ int MDCache::path_traverse(const MDRequestRef& mdr, MDSContextFactory& cf,
   if (flags & MDS_TRAVERSE_CHECK_LOCKCACHE)
     mds->locker->find_and_attach_lock_cache(mdr, cur);
 
+  if (mdr && mdr->reqid.name.is_client()) {
```

Comment:
@vshankar Only changed here. This will filter out the `client request`.

Reply:
Looks correct.